content/blog/quine-quine/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Writing a program that writes itself
+date: "2019-11-04T12:00:00.000Z"
+description: What is a Quine and how to write one.
+---
+
+Recursion is one of those concepts in programming which have always been
+incredibly fascinating to me. _What do you mean that I can call the
+function from the function itself?_
+
+Well, there is a class of programs that bring the whole idea of recursion
+to a completely different level, which is programs that are able to
+replicate themselves. These programs, named **Quines**, produce a copy of
```

Comment:
~I don't think "Quines" should be capitalised here because it's not a proper noun.~

I did some digging; see my later comment.